### ISSUE TITLE
docker build error

### ISSUE THREAD

**Issue by cmatKhan** [BergmannLab/PascalX#13]
There seems to be a dependency conflict. The dockerfile build errors with the following:

```
Installed /usr/local/lib/python3.8/dist-packages/Jinja2-3.1.2-py3.8.egg
error: docutils 0.18.1 is installed but docutils<0.18 is required by {'sphinx-rtd-theme'}
The command '/bin/sh -c cd /PascalX/python/ && python3 setup.py install' returned a non-zero code: 1
```

**Comment by sandeepacharya464** [comment]
I resolved this issue by adding "docutils<0.18" requirement in setup.py. After fixing this issue, you get similar version related error with numba and I fixed that by adding "numpy>=1.18.0, <1.23" requirement in setup.py. But the build fails even after that. 

setup(
    name='PascalX',
    description='work in progress',
    version=find_version("PascalX", "__init__.py"),
    author='D. Krefl',
    author_email='daniel.krefl@unil.ch',
    packages=find_packages(),
    url='https://github.com/BergmannLab/PascalX',
    license='LICENSE',
    long_description=open('README.md').read(),
    install_requires=[
    "docutils<0.18",
    "cffi>=1.0.0",
    "numba>=0.51.2",
    "matplotlib>=3.1.0",
    "sortedcontainers>=2.1.0",
    "tqdm>=4.43.0",
    "scipy>=1.4.0",
    "numpy>=1.18.0, <1.23",
    "fastnumbers>=3.1.0",
    "seaborn>=0.11.0",
    "progressbar>=2.5",
    "sphinx>=3.2.1",
    "sphinx-rtd-theme>=0.5.0"
    ],
    setup_requires=["cffi","path.py"],
    cffi_modules=["wchissum.py:ffibuilder","hpstats.py:ffibuilder"],
    zip_safe=False,
)


#12 72.92 Reading https://pypi.org/simple/llvmlite/
#12 73.22 No local packages or working download links found for llvmlite<0.40,>=0.39.0dev0
#12 73.22 error: Could not find suitable distribution for Requirement.parse('llvmlite<0.40,>=0.39.0dev0')
executor failed running [/bin/sh -c cd /PascalX/python/ && python3 setup.py install]: exit code: 1

Clearly, there is a lot of issue with this dockerfile build.

**Comment by sandeepacharya464** [comment]
the error with llvmlite can be fixed by adding numba < 0.52.0 requirement in setup.py. Then the build worked for me.

**Comment by Dan-RAI** [comment]
Thanks for spotting the requirement errors. Numba is currently not anymore in use and can be dropped. The .txt will be fixed asap.
